tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
-method = "dSPM"
-
-##############################################################################
-# Compute the inverse solution:
-
-stc = apply_inverse(evoked, inverse_operator, lambda2, method)
-
-##############################################################################
-# Save the source time courses to disk:
-
-stc.save('mne_dSPM_inverse')
-
-##############################################################################
-# Now, let's compute dSPM on a raw file within a label:
-
-fname_label = data_path + '/MEG/sample/labels/Aud-lh.label'
-label = mne.read_label(fname_label)
-
-##############################################################################
-# Compute inverse solution during the first 15s:
-
-from mne.minimum_norm import apply_inverse_raw  # noqa
-start, stop = raw.time_as_index([0, 15])  # read the first 15s of data
-stc = apply_inverse_raw(raw, inverse_operator, lambda2, method, label,
-                        start, stop)
-
-##############################################################################
-# Save result in stc files:
-
-stc.save('mne_dSPM_raw_inverse_Aud')
-
-##############################################################################
-# What else can you do?
-# ^^^^^^^^^^^^^^^^^^^^^
-#
-#     - detect heart beat QRS component
-#     - detect eye blinks and EOG artifacts
-#     - compute SSP projections to remove ECG or EOG artifacts
-#     - compute Independent Component Analysis (ICA) to remove artifacts or
-#       select latent sources
-#     - estimate noise covariance matrix from Raw and Epochs
-#     - visualize cross-trial response dynamics using epochs images
-#     - compute forward solutions
-#     - estimate power in the source space
-#     - estimate connectivity in sensor and source space
-#     - morph stc from one brain to another for group studies
-#     - compute mass univariate statistics base on custom contrasts
-#     - visualize source estimates
-#     - export raw, epochs, and evoked data to other python data analysis
-#       libraries e.g. pandas
-#     - and many more things ...
-#
-# Want to know more ?
-# ^^^^^^^^^^^^^^^^^^^
-#
-# Browse :ref:`the examples gallery <sphx_glr_auto_examples>`.
-
-print("Done!")
+# Finally, we can estimate the cortical origins of the evoked activity by
+# projecting the sensor data into this subject's :term:`source space`.
+# MNE-Python supports lots of ways of doing this (minimum-norm estimation,
+# dipole fitting, beamformers, etc); here we'll use minimum-norm estimation
+# (MNE) to generate a continuous cortical activation map. To do this we'll
+# need the inverse operator for this subject (the sample data includes one
+# that's been pre-computed), and we'll need to specify a regularization
+# parameter:
+
+inverse_operator_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                     'sample_audvis-meg-oct-6-meg-inv.fif')
+inv_operator = mne.minimum_norm.read_inverse_operator(inverse_operator_file)
```

Comment:
I would actually expand this step. It's a bit opaque otherwise to beginners and something I struggled with also initially. Basically, show how to compute a covariance, a source space and a forward. So, the only external files you use are the trans which comes from coregistration (and we can plot it here) and the bem (can also be plotted) which you mention that one gets from Freesurfer by running recon-all.

Reply:
Just to explain a bit further, many of the MNE examples start from these intermediate files and it's not clear how to get to those files. You could of course link to an example which produces the file, but even then the user has to follow the bread crumbs and hope that at some point he doesn't have to go to another link ...

Reply:
I agree that it would be nice to explain where that inverse operator comes from, what went into computing it. Perhaps a link to another tutorial (including BEM plots etc.) would suffice?

Reply:
This is the only comment I haven't addressed. I'd like to get a couple other opinions. My worry is that this tutorial is already quite long, and that explaining all the steps in calculating a covariance, source space, forward, and inverse are enough to fill an entire tutorial alone and would make this one *waaay* too long (note that even [this one](https://mne-tools.github.io/dev/auto_tutorials/source-modeling/plot_mne_dspm_source_localization.html) "cheats" by loading the forward from file).

@larsoner @agramfort @dengemann WDYT?  @jasmainak would you be satisfied by a new sentence in the preceding paragraph describing what steps are being skipped here (and if so could you take a first pass at writing that sentence)?

Reply:
I'm actually okay moving evoked to source space in another quickblitz tutorial, but one that does all the steps. I find skipping the forward model a little weird (also in the example you linked to above) in a source localization tutorial because almost in no circumstances can you skip it if you want to get to source space when analyzing your own data. And to someone who has processed EEG/MEG data before in another software package, they immediately know something has been skipped.